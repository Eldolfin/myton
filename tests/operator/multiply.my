print(5 * 3) # expect: 15
print(12.34 * 0.3) # expect: 3.702

# string multiplication
print("hello" * 3) # expect: hellohellohello

# list multiplication
print([1, 2, 3] * 3) # expect: [1, 2, 3, 1, 2, 3, 1, 2, 3]
