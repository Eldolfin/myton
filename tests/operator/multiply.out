15
3.702
hellohellohello
[1, 2, 3, 1, 2, 3, 1, 2, 3]
